### PR TITLE
Textutilities navbar component

### PR DESCRIPTION
**Pull request by Mohitk391**
Components added: Navigation bar, Text Utilities

Files to check out: 
--  navigation.html
--  navigation.css
--  typography.html
--  typography.css

Preview link: https://stupefied-golick-7f2a0a.netlify.app/

### CODE REVIEW

**Comment by varun-Tad** [comment]
The Code looks good to merge.You can go ahead

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                    <p>
+                        The Navigation is of basically the first component of any webpage, which consists of name/logo of the website, and some optional components like searchbar, links, etc.
+                    </p>
+                    <h3 class="component_heading">Navigation for Desktop</h3>
+                    <h4>Simple Navigation bar for desktop-</h4>
+                    <div class="component_wrapper_box">
+                        <nav class="navbar">
+                            <a href="#" class="logo">EASY UI</a>
+                            <div class="external-links">
+                                <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
```
Comment (Adarsh-Patel-Dev) [inline components/navigation/navigation.html:78]: add live links here

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    font-family: 'Poppins', sans-serif;
+    font-size: 2rem;
+    padding: 0 1.5rem; 
+    
+}
+
+.external-links {   
+    width: 20%;
+    justify-self: end;
+
```
Comment (Adarsh-Patel-Dev) [inline components/navigation/navigation.css:26]: remove spaces here

**Review by Adarsh-Patel-Dev — APPROVED** [review]
Suggested some changes 
you can merge after working on those suugestions

**Review by prankurpandeyy — COMMENTED** [review]
looks good to merge

**Pull request merged**